In [38]:
import pandas as pd

In [44]:
df = pd.read_csv("airport.csv")
df.head()

1                               Goroka Airport        Goroka  \
0  2                               Madang Airport        Madang   
1  3                 Mount Hagen Kagamuga Airport   Mount Hagen   
2  4                               Nadzab Airport        Nadzab   
3  5  Port Moresby Jacksons International Airport  Port Moresby   
4  6                  Wewak International Airport         Wewak   

   Papua New Guinea  GKA  AYGA  -6.081689834590001  145.391998291  5282  10  \
0  Papua New Guinea  MAG  AYMD           -5.207080     145.789001    20  10   
1  Papua New Guinea  HGU  AYMH           -5.826790     144.296005  5388  10   
2  Papua New Guinea  LAE  AYNZ           -6.569803     146.725977   239  10   
3  Papua New Guinea  POM  AYPY           -9.443380     147.220001   146  10   
4  Papua New Guinea  WWK  AYWK           -3.583830     143.669006    19  10   

   U  Pacific/Port_Moresby  airport  OurAirports  
0  U  Pacific/Port_Moresby  airport  OurAirports  
1  U  Pacific/Port_Moresby  airport  OurAirports  
2  U  Pacific/Port_Moresby  airport  OurAirports  
3  U  Pacific/Port_Moresby  airport  OurAirports  
4  U  Pacific/Port_Moresby  airport  OurAirports

In [45]:
df.isna().sum()

1                        0
Goroka Airport           0
Goroka                  49
Papua New Guinea         0
GKA                      0
AYGA                     0
-6.081689834590001       0
145.391998291            0
5282                     0
10                       0
U                        0
Pacific/Port_Moresby     0
airport                  0
OurAirports              0
dtype: int64

0 null values found in columns that are being used. 49 values found in the city name column but column is not being used so will not have an effect on our project goals.

In [48]:
df1 = pd.read_csv("routes.csv")
df1.head()

2B  410  AER  2965  KZN  2990 Unnamed: 6  0  CR2
0  2B  410  ASF  2966  KZN  2990        NaN  0  CR2
1  2B  410  ASF  2966  MRV  2962        NaN  0  CR2
2  2B  410  CEK  2968  KZN  2990        NaN  0  CR2
3  2B  410  CEK  2968  OVB  4078        NaN  0  CR2
4  2B  410  DME  4029  KZN  2990        NaN  0  CR2

In [47]:
df1.isna().sum()

2B                0
410               0
AER               0
2965              0
KZN               0
2990              0
Unnamed: 6    53065
0                 0
CR2              18
dtype: int64

0 null values found in columns being used. 53065 null values found in column 7 and 18 in column 9 but these columns aren't being used and so they will not have an effect on our project goals.